package.json

```diff
@@ -14,16 +14,19 @@
   },
   "scripts": {
     "install-if-package-changed": "git diff-tree -r --name-only --no-commit-id ORIG_HEAD HEAD | grep --quiet yarn.lock && yarn install || exit 0",
-    "start": "react-scripts start",
-    "start:backend": "UPDATE_DATA=true DEV=true ts-node src/backend/app.ts",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
     "prepare": "husky install",
-    "pre-commit": "concurrently 'pretty-quick --staged' 'lint-staged' 'tsc --noEmit'"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
+    "pre-commit": "concurrently 'pretty-quick --staged'",
+    "start:frontend": "cd frontend && yarn start && cd ..",
+    "start:backend": "cd backend && yarn start && cd ..",
+    "clean:frontend": "cd frontend && yarn clean && cd ..",
+    "clean:backend": "rm -rf node_modules && rm -rf build && cd backend && yarn clean && cd ..",
+    "clean": "run-p clean:frontend clean:backend",
+    "prebuild:frontend": "cd frontend && yarn install && cd ..",
+    "prebuild:backend": "cd backend && yarn install && cd ..",
+    "prebuild": "run-s prebuild:backend prebuild:frontend",
+    "build:frontend": "cd frontend && yarn build && cd ..",
+    "build:backend": "cd backend && yarn build && cd ..",
+    "build": "run-s build:frontend build:backend"
```

Comment:
is there a reason we're not using `workspaces`? It would simplify a lot of these build steps

Reply:
It took us a while to figure out how to build frontend and backend properly. Adding an extra variable in config seemed a bit too much. :sweat_smile: We should definitely try workspaces down the line. :100: